deploy/helm/sumologic/conf/setup/custom.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+export SUMOLOGIC_BASE_URL=
+export SUMOLOGIC_ACCESSKEY=
+export SUMOLOGIC_ACCESSID=
+
+# shellcheck disable=SC2010
+for dir in $(ls /customer-scripts | grep _ | grep -oE '^.*?_' | sed 's/_//g' | sort | uniq); do
```

Comment:
1.  If we're going to use `ls` for this I guess this should be `ls -1` to force one entry per line. I might be mistaken though.

```suggestion
for dir in $(ls -1 /customer-scripts | grep _ | grep -oE '^.*?_' | sed 's/_//g' | sort | uniq); do
```

2. Do you mind explaining in the comments what specifically is being done to the file names here? A particular file name format is expected?